novm/manager.py

```diff
@@ -227,7 +227,7 @@ def state(output):
                 index=1+len(nics)+len(disks),
                 pci=not(nopci),
                 tag="root",
-                tempdir=self._instances.file(str(os.getpid())),
+                tempdir=self._myInstance.file("writable"),
```

Comment:
self._myInstance is added at line 338~341, since state is a callback function, at this time, we should have _myInstance defined.